test/0.8.9/withdrawal-queue-nft.test.js

```diff
@@ -35,7 +35,7 @@ contract('WithdrawalQueue', ([owner, stranger, daoAgent, user, tokenUriManager,
     erc721ReceiverMock = await ERC721ReceiverMock.new({ from: owner })
 
     await steth.setTotalPooledEther(ETH(600))
-    // we need 1 ETH additionally to pay gas on finalization because coverage ingnores gasPrice=0
+    // we need 1 ETH additionally to pay gas on finalization because coverage ignores gasPrice=0
```

Comment:
maybe clarify a bit

it's unclear that the comment is about `solidity coverage` and not about `in-protocol coverage application` (LIP-6)

Reply:
Or better to place gasPrice:0 in options explicitly, like in the other tests